src/config.jl

```diff
@@ -39,7 +39,8 @@ function config!(
     setpropagating!(logger, propagate)
     handler = DefaultHandler(
         stdout,
-        DefaultFormatter(fmt), Dict{Symbol, Any}(:is_colorized => colorized)
+        fmt == DEFAULT_FMT_STRING ? _default_formatter : DefaultFormatter(fmt),
```

Comment:
Should we just add memoization of everything to `DefaultFormatter`?

Reply:
I don't think that's quite necessary yet, but maybe?

Reply:
It might be cleaner?

Reply:
How would that look?

Reply:
That's also wouldn't really solve the load time issue because then we'd be running the function in the `__init__`.

Reply:
> How would that look?

```julia
const _fmt_cache = Dict{String, DefaultFormatter}()
function DefaultFormatter(fmt::String)
    get!(_fmt_cache, fmt) do
        # insert current logic for constructing DefaultFormatter here
    end
end
```

Reply:
Right, but we'd still need to call `DefaultFormatter(DEFAULT_FMT_STRING)` to cache that specific default.

Reply:
Which we already do on line 63.

Reply:
Ahh, okay, I didn't realize you're still fine with leaving that call there.

Reply:
But idk `__init__` and operating at global scope is weird.

Reply:
It wouldn't be quite as fast as `hash` + `isequal` is slow compared to `==`.
But maybe it is nicer to not have this special cased.

Reply:
Opted just to remove this as most of the slowness was fixed by moving the `localzone` call to where it's actually needed. Caching this just removes some pretty quick regex parsing.